From 8b3ac2eeb4538f015be22367a9e692cf9be995cd Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 21 Oct 2015 09:40:25 +0200
Subject: Add redhat directory

Merged patches (20170123):
- 3dc49f9 Customize ROM banner timeout
- 4845fc2 ipxe: enable e1000e rom
---
 .gitpublish                        |   8 +
 redhat/.gitignore                  |   2 +
 redhat/Makefile                    |  71 ++++++++
 redhat/Makefile.common             |  35 ++++
 redhat/USAGE                       |  16 ++
 redhat/ipxe.spec.template          | 337 +++++++++++++++++++++++++++++++++++++
 redhat/rpmbuild/BUILD/.gitignore   |   1 +
 redhat/rpmbuild/RPMS/.gitignore    |   1 +
 redhat/rpmbuild/SOURCES/.gitignore |   1 +
 redhat/rpmbuild/SPECS/.gitignore   |   1 +
 redhat/rpmbuild/SRPMS/.gitignore   |   1 +
 redhat/scripts/frh.py              |  27 +++
 redhat/scripts/git-backport-diff   | 302 +++++++++++++++++++++++++++++++++
 redhat/scripts/git-compile-check   | 215 +++++++++++++++++++++++
 redhat/scripts/process-patches.sh  |  77 +++++++++
 redhat/scripts/tarball_checksum.sh |   3 +
 src/config/local/general.h         |   3 +
 17 files changed, 1101 insertions(+)
 create mode 100644 .gitpublish
 create mode 100644 redhat/.gitignore
 create mode 100644 redhat/Makefile
 create mode 100644 redhat/Makefile.common
 create mode 100644 redhat/USAGE
 create mode 100644 redhat/ipxe.spec.template
 create mode 100644 redhat/rpmbuild/BUILD/.gitignore
 create mode 100644 redhat/rpmbuild/RPMS/.gitignore
 create mode 100644 redhat/rpmbuild/SOURCES/.gitignore
 create mode 100644 redhat/rpmbuild/SPECS/.gitignore
 create mode 100644 redhat/rpmbuild/SRPMS/.gitignore
 create mode 100755 redhat/scripts/frh.py
 create mode 100755 redhat/scripts/git-backport-diff
 create mode 100755 redhat/scripts/git-compile-check
 create mode 100755 redhat/scripts/process-patches.sh
 create mode 100755 redhat/scripts/tarball_checksum.sh
 create mode 100644 src/config/local/general.h

diff --git a/src/config/local/general.h b/src/config/local/general.h
new file mode 100644
index 0000000..5814511
--- /dev/null
+++ b/src/config/local/general.h
@@ -0,0 +1,3 @@
+#undef ROM_BANNER_TIMEOUT
+#define ROM_BANNER_TIMEOUT     0
+
-- 
1.8.3.1

